packages/core/src/feature-app-manager.ts

```diff
@@ -432,10 +455,12 @@ export class FeatureAppManager {
     };
 
     try {
-      if (beforeCreate) {
-        beforeCreate(env);
-      }
+      beforeCreate?.(env);
```

Comment:
this changes the current behavior. Is this intended?

Reply:
It was, but after discussing it I agree that this should be reverted.